src/code/z_jpeg.c

```diff
@@ -23,9 +23,9 @@ void Jpeg_ScheduleDecoderTask(JpegContext* ctx) {
         0,                    // flags
         NULL,                 // ucode_boot
         0,                    // ucode_boot_size
-        gJpegUCode,           // ucode
+        njpgdspMainTextStart,           // ucode
         0x1000,               // ucode_size
-        gJpegUCodeData,       // ucode_data
+        njpgdspMainDataStart,       // ucode_data
```

Comment:
Try to keep the same indentation

Reply:
Fixed this just before your review came through 😅